Library/Homebrew/cask/test/cask/cli/reinstall_test.rb

```diff
@@ -0,0 +1,27 @@
+require "test_helper"
+
+describe Hbc::CLI::Reinstall do
+  it "allows reinstalling a Cask" do
+    shutup do
+      Hbc::CLI::Install.run("local-transmission")
+    end
+    Hbc.load("local-transmission").must_be :installed?
+    shutup do
+      Hbc::CLI::Reinstall.run("local-transmission")
+    end
+
```

Comment:
Move this empty line before the `shutup` block, so the block is grouped with the `Hbc.load` line it belongs to. The same applies below.